ts/components/settings/section/CategoryRecoveryPassword.tsx

```diff
@@ -68,9 +67,11 @@ const qrLogoProps: QRCodeLogoProps = {
 };
 
 export const SettingsCategoryRecoveryPassword = () => {
-  const [loadingSeed, setLoadingSeed] = useState(true);
-  const [recoveryPhrase, setRecoveryPhrase] = useState('');
-  const [hexEncodedSeed, setHexEncodedSeed] = useState('');
+  const recoveryPhrase = getCurrentRecoveryPhrase();
+  if (!recoveryPhrase || isEmpty(recoveryPhrase)) {
```

Comment:
I guess we don't need isEmpty anymore right?

Reply:
yep good catch. It's a string so falsy when empty